test/with_dummyserver/test_connectionpool.py

```diff
@@ -104,7 +104,7 @@ def test_timeout(self) -> None:
             delta = time.time() - now
 
             message = "timeout was pool-level SHORT_TIMEOUT rather than request-level LONG_TIMEOUT"
-            assert delta >= LONG_TIMEOUT, message
+            assert delta >= (LONG_TIMEOUT - 1e-5), message
```

Comment:
Since you mention `pytest.approx`, can we use that in this situation?

Reply:
I tried and it’s not supported for greater than : less than comparisons. It’s also mentioned in the pytest.approx documentation that is only for equality tests.